ci/common/build.sh

```diff
@@ -7,27 +7,26 @@ build_make() {
 }
 
 build_deps() {
-  if [[ "${BUILD_32BIT}" == ON ]]; then
+  if test "x${BUILD_32BIT}" = xON ; then
     DEPS_CMAKE_FLAGS="${DEPS_CMAKE_FLAGS} ${CMAKE_FLAGS_32BIT}"
   fi
-  if [[ "${FUNCTIONALTEST}" == "functionaltest-lua" ]]; then
+  if test "x${FUNCTIONALTEST}" = "xfunctionaltest-lua" ; then
```

Comment:
So I never use `[ … ]` in my shell scripts, it is misleading and harder to type, even with my “basic” mappings which include universal `,h` (`[]<left>`), but not something for `[  ]<left><left>` which is shell-specific. Either `[[` or `test`.